--- gaussian-16.C.01/g16/bsd/gau-hname	2016-08-02 06:22:04.000000000 -0700
+++ patch-files/gau-hname	2019-08-13 13:03:50.716835428 -0700
@@ -1,15 +1,14 @@
-#!/bin/csh -f
-set xname = "$g16root/g16"
-if ((-e $xname/ia32p4.flag) || (-e $xname/ia32p3.flag) ) then
-  set hname = "i386"
-else if (-e $xname/mac32.flag) then
-  set hname = "i386-mac32"
-else if ((-e $xname/em64t.flag) || (-e $xname/nehalem.flag) || (-e $xname/sandy.flag) \
-  || (-e $xname/haswell.flag)) then
-  set hname = "em64t"
-else if (-e $xname/gau-machine) then
-  set hname = `gau-machine`
+#!/bin/bash -f
+xname="$g16root/g16"
+if test -e $xname/ia32p4.flag -o -e $xname/ia32p3.flag;then 
+  hname="i386"
+elif test -e $xname/mac32.flag; then
+  hname="i386-mac32"
+elif test -e $xname/em64t.flag -o -e $xname/nehalem.flag -o -e $xname/sandy.flag -o -e $xname/haswell.flag; then
+  hname="em64t"
+elif test -e $xname/gau-machine; then
+  hname=`gau-machine`
 else
-  set hname = "unknown"
-  endif
+  hname="unknown"
+fi
 echo -n $hname
