--- a/Src/m_pulay.F90	2023-04-18 19:33:18.666313709 -0700
+++ b/Src/m_pulay.F90	2023-04-18 19:32:10.337012477 -0700
@@ -279,7 +279,8 @@
     !
     ! Perform linear mixing if iscf = N x nkick
     !
-    if (nkick > 0 .AND. mod(iscf,nkick).eq.0) then
+    if (nkick > 0) then 
+       if (mod(iscf,nkick).eq.0) then
        ! Reset the history information
        if (Node == 0) then
           write(6,"(a,i4)") "Applying kick at iscf: ", iscf
@@ -289,6 +290,7 @@
 
        call linear_mixing(alphakick)
        RETURN
+      endif
     endif
     !
     ! .......................
